coreMacros/src/main/scala/chisel3/internal/sourceinfo/NamingAnnotations.scala

```diff
@@ -82,9 +82,12 @@ class NamingTransforms(val c: Context) {
         tree  // don't recurse into inner classes
       case q"$mods def $tname[..$tparams](...$paramss): $tpt = $expr" => {
         val Modifiers(_, _, annotations) = mods
+        // don't apply naming transform twice
         val containsChiselName = annotations.map({q"new chiselName()" equalsStructure _}).fold(false)({_||_})
-        if (containsChiselName) {
-          tree  // don't apply the transform multiple times
+        // transforming overloaded initializers causes errors, and the transform isn't helpful
+        val isInitializer = tname == TermName("<init>")
+        if (containsChiselName || isInitializer) {
```

Comment:
Yeah, this caused problem in one of the classes with multiple constructors (Queue, I think).